_docs/energy_dashboard.md

```diff
@@ -12,17 +12,31 @@
 
 ### For Electricity
 
-<img src="./assets/current_consumption_electricity.png" alt="HA modal electricity example" height="500">
+You can only record current (i.e. today's) consumption in the Energy dashboard if you have a way of measuring live consumption in your home.
+
+#### Octopus Home Mini
 
-This is only available if you have an Octopus Home Mini and a smart electricity meter.
+If you have an Octopus Home Mini and a smart electricity meter you can obtain live meter reading data into Home Assistant:
 
 1. Go to your [energy dashboard configuration](https://my.home-assistant.io/redirect/config_energy/)
 2. Click `Add Consumption` under `Electricity Grid`
 3. For `consumed energy` you want `sensor.octopus_energy_electricity_{{METER_SERIAL_NUMBER}}_{{MPAN_NUMBER}}_current_consumption` or `sensor.octopus_energy_electricity_{{METER_SERIAL_NUMBER}}_{{MPAN_NUMBER}}_current_accumulative_consumption`
 4. For `Use an entity tracking the total costs` option you want `sensor.octopus_energy_electricity_{{METER_SERIAL_NUMBER}}_{{MPAN_NUMBER}}_current_accumulative_cost`
 
+<img src="./assets/current_consumption_electricity.png" alt="HA modal electricity example" height="500">
+
 > Please note that data will only appear in the energy dashboard from the point you configure the Home Mini within the integration. It doesn't backport any data.
 
+#### Alternative methods to measure current Home Consumption
+
+If you don't have an Octopus Home mini you may have another way to get live or near-live daily consumption into Home Assistant such as a Hildebrand Glow In Home Display, an Energy CT Clamp such as the Shelly EM on the incoming supply cable, or your existing Solar/Battery inverter may have a sensor that provides Grid import information that you can use in the Energy dashboard.
+
+Do be aware that as you are not directly capturing the smart meter readings in Home Assistant there will always be a small measurement difference from what Octopus energy say you have used, but in practice the difference is likely to be quite small.
+
+1. Create a utility meter that resets daily to store the consumption sensor information in, e.g. called `Grid Import Today`
+2. The utility meter should point to the sensor that is measuring your grid import. e.g. for a Hildebrand Glow it could be `sensor.smart_meter_electricity_energy_import`; a Shelly EM will be `sensor.<EM channel name>_energy_total`; for a GivEnergy inverter using the GivTCP integration it will be `sensor.givtcp_XXyywwXnnn_import_energy_today_kwh`
+3. Then add the consumption information to the Energy dashboard as per the steps for Octopus Home Mini above.  For step 3, `consumed energy`, you want the utility meter you have just created above, e.g. `sensor.grid_import_today`
```

Comment:
I think if you're using these external sensors, its probably because you don't have access to the current consumption sensors. Therefore you'd want to use the current rate sensor with the option of 'use an entity with current price'. The calculated cost will obv not include standing rate charges.

Reply:
You are absolutely correct, I had indeed been using 'entity with a current price', and in my haste to shorten the text, had overlooked that this was different from the consumption option being used for the octopus mini. Fixed now